apps/foundation-demo/src/app/product/pages/product-view/product-view.component.html

```diff
@@ -3,4 +3,7 @@
     <add-to-cart (addToCart)="ProductContainer.addToCart($event)" [additive]="ProductContainer.product$ | async"  [qty]="ProductContainer.qty$ | async"></add-to-cart>
   </product>
   <loading-icon *ngIf="(ProductContainer.loading$ | async)"></loading-icon>
+  <modal-portal>
```

Comment:
@lderrickable what happens if I want the notification on another page than "product view"? Is there a way to abstract this?

Reply:
I took `modal-portal` and moved it into `cart-modal`. Not sure what you mean by "abstract", though. It's as abstracted as I'd want it. If you want the notification on another page, you just add `cart-modal` to it.